src/decorator/idProperty.js

```diff
@@ -1,5 +1,14 @@
 import {OrmMetadata} from '../orm-metadata';
 
+/**
+ * Set the id property for en entity
+ *
+ * @export
+ * @param {string} propertyName
+ * @returns {function}
+ *
+ * @deocator
```

Comment:
that s from auto jsdocs. admitably that s not very consistent. others include @export and @memberOf. might get useful for auto doc generation (which i wanted to recheck)